qemu/tests/cfg/cpu_topology_test.cfg

```diff
@@ -17,13 +17,39 @@
             no RHEL.6 RHEL.7
             no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
             required_qemu = [8.2.0,)
+            check_core_id_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'
     variants:
         - default:
             no s390x
         - siblings_core:
-            only s390x
             RHEL.8:
                 check_siblings_cmd = "awk -F'-' '{print $2 - $1 + 1}' /sys/devices/system/cpu/cpu0/topology/core_siblings_list"
             RHEL.9:
                 check_siblings_cmd = 'cat /proc/cpuinfo |grep siblings |uniq |cut -d ":" -f 2'
-            check_core_id_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'
+        - check_core_per_sockets:
+            type = cpu_topology_details_test
+            start_vm = yes
+            check_core_per_socket_cmd = "lscpu --extended | awk 'NR>1 {print $5}'"
+            check_sockets_cmd = "lscpu | grep 'Socket(s)' |uniq |cut -d ":" -f 2"
+            s390-virtio:
+                cpu_model_flags = 'ctop=on'
+                check_sockets_cmd = "lscpu | grep 'Socket(s) per book' |uniq |cut -d ":" -f 2"
+            variants:
+                - two_cores_in_one_socket:
+                    vcpu_maxcpus = 2
+                    vcpu_sockets = 1
+                    vcpu_cores = 2
+                    s390-virtio:
+                        vcpu_drawers = 1
```

Comment:
You're not defining vcpu_books for s390x, is this intended? IIUC, you're using this to cover the avocado-vt PR that introduces it.

Reply:
yes, and vt would help calculate the value of books